etna/datasets/tsdataset.py

```diff
@@ -679,6 +679,25 @@ def regressors(self) -> List[str]:
         """
         return self._regressors
 
+    # TODO: is it ok that it includes target components and prediction intervals?
+    # TODO: don't we want to rename it into "variables" or smth like this for example?
+    # TODO: what about adding this into ts.describe/ts.info?
+    @property
+    def features(self) -> List[str]:
```

Comment:
I think we should consider returning a `Set` instead of a `List`. It seems like in most scenarios we want to work with feature sets (lookup, difference, etc.).

Reply:
I think that it contradicts to methods `def segments` and `def regressors`. I think that these kind of methods should work the same.

Reply:
With list we reduce performance in many places

Reply:
Decided to keep as List.